lib/build.js

```diff
@@ -372,6 +375,10 @@ module.exports = {
             if (atom.config.get('build.scrollOnError')) {
               this.errorMatcher.matchFirst();
             }
+            if (atom.packages.isPackageActive('linter')) {
```

Comment:
Might be better to check if `this.linterIntegration` is set here rather than the package activation, to avoid pesky race conditions.